controller/search_spaces_blackbox_test.go

```diff
@@ -71,20 +74,81 @@ func (rest *TestSearchSpacesREST) TestSpacesSearchOK() {
-		{"with different values", args{offset("0"), limit(10), prefix + "TEST"}, expects{differentValues()}},
+		{"With uppercase fullname query", args{ptr.String("0"), ptr.Int(10), prefix + "TEST_AB", nil}, expects{totalCount(1)}},
+		{"With lowercase fullname query", args{ptr.String("0"), ptr.Int(10), prefix + "TEST_AB", nil}, expects{totalCount(1)}},
+		{"With uppercase description query", args{ptr.String("0"), ptr.Int(10), "DESCRIPTION FOR " + prefix + "TEST_AB", nil}, expects{totalCount(1)}},
+		{"With lowercase description query", args{ptr.String("0"), ptr.Int(10), "description for " + prefix + "test_ab", nil}, expects{totalCount(1)}},
+		{"with special chars", args{ptr.String("0"), ptr.Int(10), "&:\n!#%?*", nil}, expects{totalCount(0)}},
+		{"with * to list all", args{ptr.String("0"), ptr.Int(10), "*", nil}, expects{totalCountAtLeast(len(idents))}},
+		{"with multi page", args{ptr.String("0"), ptr.Int(10), prefix + "TEST", nil}, expects{hasLinks("Next")}},
+		{"with last page", args{ptr.String(strconv.Itoa(len(idents) - 1)), ptr.Int(10), prefix + "TEST", nil}, expects{hasNoLinks("Next"), hasLinks("Prev")}},
+		{"with different values", args{ptr.String("0"), ptr.Int(10), prefix + "TEST", nil}, expects{differentValues()}},
+
+		{
+			name: "sorted by name ascending",
+			args: args{
+				pageOffset: ptr.String("0"),
+				pageLimit:  ptr.Int(10),
+				q:          prefix + "TEST",
+				sort:       ptr.String("name"),
+			},
+			expects: expects{sortedSpaces(prefix+"TEST_0", prefix+"TEST_1")},
+		},
+		{
+			name: "sorted by name descending",
+			args: args{
+				pageOffset: ptr.String("0"),
+				pageLimit:  ptr.Int(10),
+				q:          prefix + "TEST",
+				sort:       ptr.String("-name"),
+			},
+			expects: expects{sortedSpaces(prefix+"TEST_C", prefix+"TEST_B")},
+		},
+		{
+			name: "sorted by owner ascending",
+			args: args{
+				pageOffset: ptr.String("0"),
+				pageLimit:  ptr.Int(10),
+				q:          prefix + "TEST",
+				sort:       ptr.String("owner"),
+			},
+			expects: expects{sortedSpaces(prefix+"TEST_AB", prefix+"TEST_B")},
+		},
+		{
+			name: "sorted by owner descending",
+			args: args{
+				pageOffset: ptr.String("0"),
+				pageLimit:  ptr.Int(10),
+				q:          prefix + "TEST",
+				sort:       ptr.String("-owner"),
+			},
+			expects: expects{sortedSpaces(prefix+"TEST_AB", prefix+"TEST_B")},
+		},
+		{
+			name: "sorted by creation timestamp ascending",
+			args: args{
+				pageOffset: ptr.String("0"),
+				pageLimit:  ptr.Int(10),
+				q:          prefix + "TEST",
+				sort:       ptr.String("created"),
+			},
+			expects: expects{sortedSpaces(prefix+"TEST_A", prefix+"TEST_AB")},
+		},
+		{
+			name: "sorted by creation timestamp descending",
+			args: args{
+				pageOffset: ptr.String("0"),
+				pageLimit:  ptr.Int(10),
+				q:          prefix + "TEST",
+				sort:       ptr.String("-created"),
+			},
+			expects: expects{sortedSpaces(prefix+"TEST_19", prefix+"TEST_18")},
```

Comment:
It would be nice if you can add a test that passes `ptr.String("")` or `ptr.String("nonesense")` as a sort parameter, just to have a consistent error behavior.